components/button/button-icon.js

```diff
@@ -17,6 +17,12 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 
 	static get properties() {
 		return {
+			/**
+			 * A description to be added to the button for accessibility when text on button does not provide enough context
+			 * @type {string}
+			 */
+			description: { type: String },
```

Comment:
`<d2l-button-icon>` didn't have a `description`, so I've added it here for consistency since technically it could be useful.